client/my-sites/people/people-list-section-header/index.jsx

```diff
@@ -66,27 +53,10 @@ class PeopleListSectionHeader extends Component {
 			<SectionHeader className={ classes } count={ count } label={ label }>
 				{ children }
 				{ siteLink && (
-					<ButtonGroup>
-						<Button
-							compact
-							href={ siteLink }
-							className="people-list-section-header__add-button"
-							onMouseEnter={ this.showAddTooltip }
-							onMouseLeave={ this.hideAddTooltip }
-							ref="addPeopleButton"
-							aria-label={ translate( 'Invite user', { context: 'button label' } ) }
-						>
-							<Gridicon icon="plus-small" size={ 18 } />
-							<Gridicon icon="user" size={ 18 } />
-							<Tooltip
-								isVisible={ this.state.addPeopleTooltip }
-								context={ this.refs && this.refs.addPeopleButton }
-								position="bottom"
-							>
-								{ translate( 'Invite user', { context: 'button tooltip' } ) }
-							</Tooltip>
-						</Button>
-					</ButtonGroup>
+					<Button compact href={ siteLink } className="people-list-section-header__add-button">
+						<Gridicon icon="user-add" />
+						{ translate( 'Invite', { context: 'button label' } ) }
```

Comment:
This should probably be a little bit [more descriptive as to intent](https://lamda.blog/2018/03/02/psa-double-check-your-strings-for-ambiguous-contexts/). Maybe `Button to invite more users`.